src/components/ImageViewer.ts

```diff
@@ -46,7 +47,7 @@ class ImageViewer extends Component<ImageViewerProps, ImageViewerState> {
                 ref: this.props.getRef
             },
             DOM.img({
-                onClick: this.toggleLightBox,
+                onClick: this.props.onClickOption === "openFullScreen" ? this.toggleLightBox : this.props.onClick,
                 src: this.props.imageUrl,
```

Comment:
I think having a default /fallback image might confuse the users.  they might think the image has loaded when it has not.